dev/src/codewind/project/Project.ts

```diff
@@ -565,17 +557,16 @@ export default class Project implements vscode.QuickPickItem {
     }
 
     public get appMonitorUrl(): string | undefined {
-        const appMetricsPath = langToPathMap.get(this.type.language);
-        const supported = appMetricsPath != null && this.capabilities.metricsAvailable;
+        const isSupportedLanguage = ['nodejs'].includes(this.type.language);
+        const supported = isSupportedLanguage && this.capabilities.metricsAvailable;
         Log.d(`${this.name} supports metrics ? ${supported}`);
         if (!supported || !this.appUrl) {
             return undefined;
         }
-        let monitorPageUrlStr = this.appUrl.toString();
-        if (!monitorPageUrlStr.endsWith("/")) {
-            monitorPageUrlStr += "/";
-        }
-        return monitorPageUrlStr + appMetricsPath + "/?theme=dark";
+        const pfeOrigin = this.connection.url.toString();
```

Comment:
Thinking again, I think this will fail in the theia (Che) case, I think we have to use this logic:

https://github.com/eclipse/codewind-vscode/blob/master/dev/src/command/project/OpenPerfDashboard.ts#L32